frontend/src/components/Menu/NavbarNew.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <div class="auth-header position-sticky">
+    <b-navbar toggleable="lg" class="pr-4">
+      <b-navbar-brand>
+        <b-img
+          class="imgLogo position-absolute ml--3 mt-lg--2 mt-3 p-2 zindex1000"
+          :src="logo"
+          width="200"
+          alt="..."
+        />
+        <b-img
+          class="imgLogoBack mt--3 ml--3"
+          src="/img/template/gradido_background_header.png"
+          width="230"
+          alt="start background image"
+        ></b-img>
+      </b-navbar-brand>
+      <b-img class="sheet-img position-absolute zindex1000" :src="sheet"></b-img>
+
+      <b-collapse id="nav-collapse" is-nav class="ml-5">
+        <b-navbar-nav class="ml-auto" right>
+          <div class="mb-2">
+            <router-link to="/profile">
+              <div>
+                <div>
+                  <b-avatar
+                    button
+                    variant="primary"
+                    :text="avatarShortName"
+                    class="align-baseline"
+                  ></b-avatar>
+                  {{ avatarLongName }}
+
+                  <div class="text-right">
+                    <small>{{ balance | GDD }}</small>
+                  </div>
+                </div>
+              </div>
+            </router-link>
+          </div>
+        </b-navbar-nav>
+      </b-collapse>
+    </b-navbar>
+    <div class="alertBox">
+      <b-alert show dismissible variant="light" class="nav-alert text-dark">
+        <small>{{ $t('1000thanks') }}</small>
+      </b-alert>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'AuthNavbar',
+  props: {
+    balance: { type: Number, required: true },
+  },
+  data() {
+    return {
+      logo: '/img/brand/green.png',
+      sheet: '/img/template/Blaetter.png',
+    }
+  },
+  computed: {
+    avatarShortName() {
+      return this.$store.state.firstName[0] + this.$store.state.lastName[0]
+    },
+    avatarLongName() {
+      return `${this.$store.state.firstName} ${this.$store.state.lastName}`
+    },
+  },
+}
+</script>
+
+<style lang="scss">
```

Comment:
whats with all the style? Put that in the scss?

Reply:
yes, as far as possible and wise, i will put this in the scss. with a few, it doesn't make so much sense.